src/utils/clients.py

```
@@ -76,6 +76,27 @@
+    ("Dream", settings.DREAM),
+]
+
+for component_name, component_settings in BACKUP_PROVIDERS:
+    if (
+        hasattr(component_settings, "BACKUP_PROVIDER")
+        and component_settings.BACKUP_PROVIDER is not None
+        and component_settings.BACKUP_PROVIDER not in CLIENTS
+    ):
+        raise ValueError(
+            f"Backup provider for {component_name} is set to {component_settings.BACKUP_PROVIDER}, "
+            + "but this provider is not initialized. Please set the required API key/URL environment "
```

Ruff (0.14.7)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)